terraform-aws-github-runner/modules/runners/scale-down.tf

```diff
@@ -44,7 +44,44 @@ resource "aws_lambda_function" "scale_down" {
       REDIS_LOGIN                     = var.redis_login
       SCALE_DOWN_CONFIG               = jsonencode(var.idle_config)
       SECRETSMANAGER_SECRETS_ID       = var.secretsmanager_secrets_id
-
+      AWS_REGIONS_TO_VPC_IDS                = join(
```

Comment:
Could you pls run `terraform fmt -recursive`

Reply:
Sure!

This will trigger LOTS of unrelated chages, but this is OK :)